google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -514,6 +514,15 @@ def file path, generation: nil, encryption_key: nil
         # @param [Hash] metadata A hash of custom, user-provided web-safe keys
         #   and arbitrary string values that will returned with requests for the
         #   file as "x-goog-meta-" response headers.
+        # @param [Symbol, String] storage_class Storage class of the file.
+        #   Determines how the file is stored and determines the SLA and the
+        #   cost of storage. Values include `:multi_regional`, `:regional`,
+        #   `:nearline`, `:coldline`, `:standard`, and `:dra` (Durable Reduced
```

Comment:
The API does support changing to `STANDARD`, correct? I only ask because standard isn't a "real" storage class, so it might be worth testing...

> Cloud Storage offers four storage classes: Multi-Regional Storage, Regional Storage, Nearline Storage, and Coldline Storage.